utils/standalone-wizard/wizard.py

```diff
@@ -171,17 +452,33 @@ def _generate_secrets(config: 'SetupConfig'):
     save_local_config(config)
 
 
-def get_admin_auth_token(password: str) -> str:
-    response = Requests.post('http://127.0.0.1:7103/dashboard/api/v1/users/login', json={
+def login_first_time(config: 'SetupConfig', lifecycle_url: str):
+    try:
+        auth_token = get_admin_auth_token('admin', lifecycle_url)
+        logger.info('Changing default admin password…')
+        change_admin_password(auth_token, 'admin', config.admin_password, lifecycle_url)
+    except ResponseError as e:
+        if not e.status_code == 401:  # Unauthorized
+            raise e
```

Comment:
Nitpick, it could be written more succintly as `if e.status_code != 401`. But I wonder what is the intention here, because if it's 401, then the response error won't be re-raised. So the further functions will be called, but config.admin_password and possibly also config.admin_auth_token are wrong, so they won't work?

Reply:
This is to change the default superuser password, but only for the first time. 
So I'm trying to log in with `admin:admin`. If it succeeds, I need to change the password to something more secure. Otherwise, in case of 401 Unauthorized, that means password is already changed, and don't need to do anything.
`config.admin_password` is not the actual password but the one that is supposed to be set.
I'll add a comment.